pkg/source/markdown.go

```diff
@@ -118,12 +139,22 @@ func (m *markdown) parse(filename string, opts Opts) (err error) {
 			return
 		}
 
-		if count == 1 && !doxIdFound {
-			found, err := fmt.Sscanf(line, m.escape(doxIdFmt), &m.id)
+		if count == 1 || count == 2 {
```

Comment:
`if count < 3`? Or maybe just remove the condition?

Reply:
Before changing this, let me comment on something else more generally.

Reply:
Removing the condition would probably have the same effect as changing it, so I would prefer removing it.

Reply:
- [ ] remove count condition